### PR TITLE
Blog App: Creating Data Model

### PR DESCRIPTION
**Pull request by kingqabil**
- Built project schema.
- Created and run the necessary migration files.
- Table and column names match the ERD diagram.
- Foreign keys are included.
- All columns that are foreign keys have a corresponding index.

### CODE REVIEW

File: db/migrate/20220214195216_create_comments.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+
+      t.timestamps
+    end
```
Comment (bimbolabuari) [inline db/migrate/20220214195216_create_comments.rb]: According to the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) diagram, the `comment` table has a `text` column. Kindly add this.

`
t.text :text
`

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @kingqabil, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Add a migration file for creating `users`, `post`, `comment` and `likes` table

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220214195216_create_comments.rb (hunk 1 above)
Comment (kingqabil) [reply db/migrate/20220214195216_create_comments.rb]: Thanks for your suggestion

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi @kingqabil, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**